application/controllers/items.py

```diff
@@ -87,6 +89,20 @@
     location=['json'],
     help='charity parameter is required'
 )
+item_post_parser.add_argument(
+    'charity_url',
+    type=str,
+    required=True,
+    location=['json'],
+    help='charity_url parameter is required'
+)
+item_post_parser.add_argument(
+    'charity_score',
+    type=str,
```

Comment:
May be a conflict -- `charity_score` here is a string and and integer in the model

Reply:
Thanks! new changes have been pushed up